cypress/e2e/firstTests.cy.js

```diff
@@ -0,0 +1,49 @@
+/// <reference types="cypress" />
+
+const faker = require("faker"); 
+
+let username = faker.name.findName()
+let email = faker.internet.email()
+let password = faker.internet.password();
+
+describe('User should be able', () => {
+  
+  
+  before (() => {
```

Comment:
You can change before to beforeEach so you don't need to reload page in each test